drivers/regulator/Kconfig.axp192

```diff
@@ -4,7 +4,7 @@
 config REGULATOR_AXP192
 	bool "X-Power AXP192 PMIC regulator driver"
 	default y
-	depends on DT_HAS_AXP192_REGULATOR_ENABLED
+	depends on DT_HAS_X_POWERS_AXP192_REGULATOR_ENABLED
```

Comment:
This change should be applied as a fixup to your original changes, not an additional commit.

Reply:
Used a force-push, as stated in the contribution guidelines for the latest corrections.